files/default/ndb_templates/ndb_config.ini

```diff
@@ -15,71 +15,16 @@
 # On many Linux systems, you can do this by setting IRQBALANCE_BANNED_CPUS in /etc/sysconfig/irqbalance to 0xFFFFF0, and by using the isolcpus boot option in grub.conf. 
 # For specific information, see your operating system or platform documentation.
 
-MaxNoOfExecutionThreads = 8
-
-
-
-# Valid values: 4, 8, 16, 32. For 8 ldm threads, increase to 8. For 16 ldm threads, increase to 16.
-NoOfFragmentLogParts=4
-NoOfFragmentLogFiles=16
-
-#The size of a Redo Log file on disk. Read: https://mikaelronstrom.blogspot.com/2018/08/more-automated-control-in-mysql-cluster.html
-FragmentLogFileSize=16M
-
 #The number of copies of the data stored on different nodes in the cluster
 NoOfReplicas={{.Configuration.NdbdDefault.ReplicationFactor}}
 
-#The amount of main memory (RAM) used to store columns and ordered indexes in tables, plus some overhead
-
-
-DataMemory=512M
-
 # Setting this parameter to TRUE or 1 binds IP_ADDR_ANY so that connections can be made from anywhere (for autogenerated connections). The default is FALSE (0).
 TcpBind_INADDR_ANY=FALSE
 
-# maximum number of attributes, tables, indexes, and trigger objects used by indexes, events, and replication between clusters
-MaxNoOfAttributes=5000
-
-# A table object is allocated for each table and for each unique hash index in the cluster.
-MaxNoOfTables=4096
-
-MaxNoOfOrderedIndexes=2048
-
-MaxNoOfUniqueHashIndexes=512
-
-MaxDMLOperationsPerTransaction=4294967295
```

Comment:
This parameter "MaxDMLOperationsPerTransaction" was to set to high number because HW needed it. Does it also increase automatically

Reply:
I checked the documentation and the default value is the max which is 4294967295. **But** it can not be greater than `MaxNoOfConcurrentOperations` - so we are limited by that which hasn't changed. In any case @mronstro will look to improve these configuration parameters. For the moment we can go with this.

https://dev.mysql.com/doc/refman/8.0/en/mysql-cluster-ndbd-definition.html#ndbparam-ndbd-maxdmloperationspertransaction